tests/conftest.py

```diff
@@ -566,6 +560,10 @@ def _set_enrichment_options() -> None:
             application_metadata=True,
         )
     )
+
+    ApplicationIdentifiers.init_java(
+        JattachJcmdRunner(stop_event=Event(), jattach_timeout=AsyncProfiledProcess._JATTACH_TIMEOUT)
```

Comment:
```suggestion
        JattachJcmdRunner(stop_event=Event(), jattach_timeout=AsyncProfiledProcess._DEFAULT_JATTACH_TIMEOUT)
```

Let's name it that way